studio/app/common/dataclass/utils.py

```diff
@@ -18,5 +21,9 @@ def create_images_list(data):
 
 
 def save_thumbnail(plot_file):
-    # Note: In the barebone-studio version, it does nothing.
-    pass
+    plot_img = cv2.imread(plot_file)
```

Comment:
In barebone-system, since there is no need to create thumbnail images, please keep the original code (PASS without doing anything) here.

- Because of the above, barebone-studio deems opencv-python-install unnecessary at this time.
- The code for thumbnail generation will incorporate the optinist-for-server existing thumbnail generation code when merging the contents of this PR into the optinist-for-server repository.